14LessonFourteen/in-class-project/css/style.css

```diff
@@ -0,0 +1,256 @@
+.nav {
+
+}
+.header {
+    color: white;
+    padding: 40px;
+    width: 100%auto;
+}
+h2 {
+    float: left;
+    margin-top: -15px;
+    font-weight: lighter;
+    font-size: 30px;
+}
+.header a {
+    float: right;
+    margin-right: 25px;
+    margin-top: -8px;
+    padding-left: 20px;
+    text-decoration: none;
+    color: white;
+}
+.hero {
+    background-image: url("../img/wine-bg.jpg");
```

Comment:
be careful with your [relative path notation](https://css-tricks.com/quick-reminder-about-file-paths/). The way you have this path prefixed (`"../`) is pointing the browser to whatever folder is one step above `/img/`. That's why your images aren't loading. Use `"./` instead.